src/FunderMaps.Core/Services/ProductService.cs

```diff
@@ -147,18 +154,15 @@ public virtual Task<IEnumerable<AnalysisProduct>> GetAnalysisInFenceAsync(Guid u
         /// <param name="userId">Internal user id.</param>
         /// <param name="productType"><see cref="StatisticsProductType"/></param>
         /// <param name="neighborhoodCode">Neighborhood code.</param>
-        /// <param name="navigation"><see cref="INavigation"/></param>
         /// <param name="token"><see cref="CancellationToken"/></param>
         /// <returns><see cref="StatisticsProduct"/></returns>
-        public virtual async Task<StatisticsProduct> GetStatisticsByNeighborhoodAsync(Guid userId, StatisticsProductType productType, string neighborhoodCode, INavigation navigation, CancellationToken token = default)
+        public virtual async Task<StatisticsProduct> GetStatisticsByNeighborhoodAsync(
+            Guid userId,
```

Comment:
Statistic products should also be checked against user id. I suggest making an issue for this matter and leaving it as is for now, agreed?

Reply:
I'll add thi